staging/src/k8s.io/apimachinery/pkg/api/apitesting/roundtrip/compatibility.go

```diff
@@ -232,11 +232,7 @@ func CompatibilityTestFuzzer(scheme *runtime.Scheme, fuzzFuncs []interface{}) *f
 			field := metav1.ManagedFieldsEntry{}
 			c.Fuzz(&field)
 			if field.Fields != nil {
-				for k1 := range field.Fields.Map {
-					for k2 := range field.Fields.Map[k1].Map {
-						field.Fields.Map[k1].Map[k2] = metav1.Fields{}
-					}
-				}
+				field.Fields.Raw = []byte("{}")
```

Comment:
it looks like this produces no output in the test fixtures.

previously, we serialized this: `"fields": {"18":{"19":null}}`

now, `"fields"` is omitted entirely. that means if we break decoding this new format, our fixtures won't catch it.

Reply:
I'll make a follow up for this